app/tray/Account/Requests/SignPermitRequest/index.js

```diff
@@ -13,44 +13,18 @@ import TypedSignatureOverview from '../../../../../resources/Components/SimpleTy
 import { getSignatureRequestClass } from '../../../../../resources/domain/request'
 import useCopiedMessage from '../../../../../resources/Hooks/useCopiedMessage'
 
-const getPermit = (req) => {
-  const {
-    typedMessage: {
-      data: {
-        message: { deadline, spender, value, owner, nonce },
-        domain: { verifyingContract, chainId, name, version },
-        types
-      }
-    }
-  } = req
-
-  return {
-    deadline,
-    spender,
-    value,
-    owner,
-    verifyingContract,
-    chainId,
-    name,
-    nonce,
-    version,
-    types
-  }
-}
-
-const PermitOverview = ({ permit, req, chainData, originName, tokenData }) => {
-  const { owner, deadline, spender, value } = permit
+const PermitOverview = ({ req, chainData, originName }) => {
+  const { permit, tokenData, handlerId, ensDomains = {} } = req
```

Comment:
what's the type of `req` here?

Reply:
ok thought so. I don't see `ensDomains` on that type, just make sure these are consistent so we don't get any bugs down the line